kuma/wiki/jinja2/wiki/document.html

```diff
@@ -25,9 +25,9 @@
   {% set approvals_html = get_approvals_html(document, request.user) %}
 {% endif %}
 
-{% set zone_stack = document.zone_stack %}
+{% set zone_stack = document.zone_stack or document.default_zone_stack %}
 {% set is_zone = zone_stack|length %}
-{% set is_zone_root = is_zone and zone_stack[0].document == document %}
+{% set is_zone_root = is_zone and (zone_stack[-1].document in (document, document.parent)) %}
```

Comment:
I think the old way (index ``[0]`` versus ``[-1]``)was right.  Here's a zone stack w/ 2 zones:

* [Archive](https://developer.mozilla.org/en-US/docs/Archive) - Archive Zone
* [Archive/B2G_OS](https://developer.mozilla.org/en-US/docs/Archive/B2G_OS) - B2G zone
* [Archive/B2G_OS/Add-ons](https://developer.mozilla.org/en-US/docs/Archive/B2G_OS/Add-ons) - B2G Zone subpage

On Archive/B2G_OS/Add-ons, it detects that it is not the zone root, and doesn't apply the ``zone-landing`` class.

On Archive, it would detect that it is the zone root, and applies the ``zone-landing`` class.

On Archive/B2G_OS, the behavior would change.  Before, it would get the ``zone-landing`` class, and after this it would not.